### PR TITLE
Fixes #225: Improve the UI/UX of options page

### LINKED ISSUE #225: Improve the UI/UX of options page
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
The UI doesn't look that good and bootstrap hasn't been integrated correctly.

**Expected behavior:**
Good looking UI with Bootstrap

**Steps to reproduce:**
Open Options page

**System information:**
```
Arch Linux
```

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by akshatnitd**
Fixes #225 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
The UI of the options page isn't uniform and is not propoerly integrated with Bootstrap

#### Changes proposed in this pull request:

- Improve the UI of options page
- Integrate Bootstrap

#### Screenshot:
screenshot from 2017-12-24 01-41-25

### CODE REVIEW

**Comment by akshatnitd** [comment]
@ms10398 @gabru-md @zamhaq Please review :)

File: src/css/options.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+.container{
+	margin-top: 8%;
+}
+
+.card {
+    background-color: #0084ff;
+    padding: 40px 0px 20px;
+    box-shadow: 2px 2px 5px rgba(0,0,0,0.5);
+	-webkit-box-shadow:2px 2px 5px rgba(0,0,0,0.5);
```
Comment (zamhaq) [inline src/css/options.css]: fix indentation

File: src/css/options.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	-moz-box-shadow:2px 2px 5px rgba(0,0,0,0.5);
```
Comment (zamhaq) [inline src/css/options.css]: Not really needed. Chrome extensions won't run in Mozilla.

**Comment by akshatnitd** [comment]
@zamhaq Made the suggested changes :)

**Comment by akshatnitd** [comment]
@ms10398 @zamhaq @gabru-md Can you please review it ? Some issues are dependent on it.

**Comment by zamhaq** [comment]
@stealthanthrax @ms10398 @gabru-md Please review.

**Pull request merged**